simplq/src/components/pages/Admin/index.jsx

```diff
@@ -78,7 +80,7 @@ export default (props) => {
   const Navbar = () => (
     <div>
       <nav className={styles['navbar']}>
-        <img src="public/LogoLight.png" alt="Home" />
+        {/* <img src="public/LogoLight.png" alt="Home" /> */}
         {/* <p className={styles['simplq']}>SimplQ</p>
         <p className={styles['sign-in']}>Sign In / Sign Up</p> */}
```

Comment:
Let's not have the sign in sign up for now. We can add it once we add auth.

Also, why did you comment out he logo?

Reply:
I'll remove the navbar for now. I don't know why, but the logo image never works for me.

Reply:
Change to `../../public/LogoLight.png` or maybe add a slash:
```
/public/LogoLight.png
```

It's happening because it's a relative path to the current `window.location`

Reply:
Fixed this now. It should be /LogoLight.png